### PR TITLE
Agility plugin mark of grace overlay color change and toggle

### PR DESCRIPTION
**Pull request by HSJ-OSRS**

### CODE REVIEW

**Comment by deathbeam** [comment]
Why not just disable the coloring when highlight marks is disabled

**Comment by HSJ-OSRS** [comment]
im not a fan of the overlay colour change but i do like the little square around the marks

**Comment by deathbeam** [comment]
You can then highlight the marks with ground items plugin for same functionality imo.

File: runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityConfig.java [hunk 1]
```diff
@@ -91,11 +91,22 @@ default Color getMarkColor()
 		return Color.ORANGE;
 	}
 
+	@ConfigItem(
+		keyName = "getOverlayColorMark",
+		name = "Overlay Color Mark Of Grace",
+		description = "Color of the agility overlay with a Mark of Grace spawned",
+		position = 6
+	)
+	default Color getOverlayColorMark()
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityConfig.java]: There is already getMarkColor... isn't that sufficient?

**Pull request closed without merging**